index.html

```diff
@@ -158,41 +158,51 @@ <h4>Profesional Art Printing Data More</h4>
     </div>
     <!------------------about me------------------------------------------------>
     <div class="aboutme-container">
-                <div class="aboutme-left">
-                <h4>About me</h4>
-                <p>Hey there! I can help you build a product, feature or website. Look through some of my work and
-                    experience! If you like
-                    what you see and have a project you need coded, don’t hesitate to contact me.</p>
-            </div>
-            <div class="aboutme-right">
```

Comment:
- [ ] In the projects section the design with a cursor pointer is representing the state of the individual project on hover. We are supposed to make it such that the dark opacity disappears when hovering over it and the image with the `See Project` button appears. Please update them accordingly. 🤓️

| Normal state                                                             | Hover state                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/214768181-8b4a0ad9-cf8b-4f28-a299-f1a7b5782152.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/214768176-dc3f056b-78bb-4e36-be88-cd940bb53f48.png" width="300">|